build/internal/CreateServiceManifest.ps1

```diff
@@ -63,6 +63,13 @@ foreach ($file in $filesToAdd)
     $newElement = $templateXml.CreateElement("file", $wmxmlNs)
     $newElement.SetAttribute("destinationDir", "$fileDestinationDir")
     $newElement.SetAttribute("source", "$source")
+    if ($source.EndsWith(".resources.dll"))
```

Comment:
might be better to double check if the name already exists but totally okay with this too.

Reply:
Agree, i'm just moving quick here. I'll add a TODO: at least and we can consider logging an issue.

Reply:
nvm stopped being lazy. fixed.